scripts/experiment_2/nbconverted/4_similarity_analysis_hausdorff.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Similarity analysis
+# 
+# We want to determine if the different batch simulated data is able to capture the biological signal that is present in the original data:  How much of the real input data is captured in the simulated batch data?
+# 
+# In other words, we want to ask: “do these datasets have similar patterns”?
+# 
+# To do this we will use [Hausdorff distance](https://en.wikipedia.org/wiki/Hausdorff_distance).
```

Comment:
I could be totally misunderstanding how it's implemented in scipy, but isn't hausdorff assuming each point of space is corresponding to the same location of the other space? since batch is added at random for each data +/- for each batch scheme, that wouldn't be good, right?

Reply:
My understanding from the implementation is that hausdorff is performing all pairwise distances and looking for the max distance between points.  The data actually gets shuffled, so I do not think there is a sample-to-sample distance.